### PR TITLE
Expand installation and contribution docs

### PR DESCRIPTION
**Pull request by Midnighter**
I've improved the installation and contributing guidelines as commented in the pyOpenSci review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,18 @@ It's as simple as:
...
+```
+
+and thus automatically generated
+[Docker](https://quay.io/repository/biocontainers/taxpasta?tab=tags) and
+[Singularity](https://depot.galaxyproject.org/singularity/)
+[BioContainers](https://biocontainers.pro/) images also exist.
+
 ## Copyright
 
 -   Copyright © 2022, Moritz E. Beber, Maxime Borry, James A. Fellows Yates, and
```
Comment (sofstam) [inline README.md:50]:
```suggestion
-   Copyright © 2023, Moritz E. Beber, Maxime Borry, James A. Fellows Yates, and
```

**Review by sofstam — APPROVED** [review]
Lgtm, one minor comment :)

File: README.md (hunk 1 above)
Comment (Midnighter) [reply README.md:50]: Maybe we should add a date range then. The first code was written in 2022 so that's when it first applied.

Comment (sofstam) [reply README.md:50]: Ah I see.

**Comment by Midnighter** [comment]
Based on comment from https://github.com/pyOpenSci/software-submission/issues/84

**Pull request merged**